src/v2/Apps/NewForYou/NewForYouApp.tsx

```diff
@@ -1,23 +1,33 @@
 import React, { FC } from "react"
 import { MetaTags } from "v2/Components/MetaTags"
-import { Spacer, Text } from "@artsy/palette"
+import { Message, Spacer, Text } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { NewForYouApp_viewer } from "v2/__generated__/NewForYouApp_viewer.graphql"
 import { NewForYouArtworksGridFragmentContainer } from "v2/Apps/NewForYou/Components/NewForYouArtworksGrid"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { useSystemContext } from "v2/System"
 
 interface NewForYouAppProps {
   viewer: NewForYouApp_viewer
 }
 
 export const NewForYouApp: FC<NewForYouAppProps> = ({ viewer }) => {
+  const { isLoggedIn } = useSystemContext()
   return (
     <>
       <Spacer mt={2} />
       <MetaTags title="New For You" />
       <Text variant="xl" mt={4}>
         New Works For You
       </Text>
-      <Spacer mt={4} />
+      <Spacer mt={2} />
+      {!isLoggedIn && (
+        <Message variant="warning">
+          Already have an account? <RouterLink to="/login">Log in</RouterLink>{" "}
+          to see your personalized recommendations.
+        </Message>
+      )}
+      <Spacer mt={2} />
```

Comment:
I still think both of these `Spacer`s should be nested within the conditionals? For instance if you're logged in you get a stack of size 2 Spacers? I don't know what this should be exactly but it seems like it'd be weirdly inconsistent depending on login/out and whether or not there is data.

Reply:
Fixed